shared/teams/team/container.js

```diff
@@ -49,6 +50,17 @@ const mapStateToProps = (state: TypedState, {routeProps, routeState}): StateProp
     ['teams', 'teamNameToImplicitAdminUsernames', teamname],
     I.Set()
   )
+  // We had to request every subteam of the top-level team, rather than just
+  // child subteams of the subteam we care about.  Here's where we fix that up.
+  const subteams = state.entities
```

Comment:
did we talk to core about just getting a new rpc that does this?

Reply:
Good idea, filed CORE-6978 for the missing Gregor category for the subteam banner being shown, and CORE-6983 for this teamTree RPC accepting starting from a subteam.